### ISSUE TITLE
The abnormal loss of FasterRCNN

### ISSUE THREAD

**Issue by REREREGO** [Chasel-Tsui/mmdet-rfla#77]
Hello, when I used the configuration file you provided to train the fasterRCNN on the AITOD dataset, the training loss was nan. Could you please tell me how to solve this problem?

<img width="1813" height="239" alt="Image" src="https://github.com/user-attachments/assets/fa7b76f4-b7d4-4dbb-9577-b3fd56a54aea" />

**Comment by Chasel-Tsui** [comment]
Hi, thanks for your question. sometimes it happens because the learning rate is too high, or the warm-up iterations are small, so that the training is not stable at the early stage, a simple solution would be to tune down the learning rate and tune up the warm-up iterations a bit to see if it changes
